code/modules/mob/mob_helpers.dm

```diff
@@ -256,19 +256,20 @@
 	for(var/datum/multilingual_say_piece/S in message_pieces)
 		S.message = stars(S.message, pr)
 
-/proc/slur(phrase, list/slurletters = ("'"))//use a different list as an input if you want to make robots slur with $#@%! characters
+/proc/slur(phrase, list/slurletters = ("'"), skip_replacement = FALSE) //use a different list as an input if you want to make robots slur with $#@%! characters
 	phrase = html_decode(phrase)
 	var/leng=length(phrase)
 	var/counter=length(phrase)
 	var/newphrase=""
 	var/newletter=""
 	while(counter>=1)
 		newletter=copytext(phrase,(leng-counter)+1,(leng-counter)+2)
-		if(rand(1,3)==3)
-			if(lowertext(newletter)=="o")	newletter="u"
-			if(lowertext(newletter)=="s")	newletter="ch"
-			if(lowertext(newletter)=="a")	newletter="ah"
-			if(lowertext(newletter)=="c")	newletter="k"
+		if(!skip_replacement)
+			if(rand(1,3)==3)
+				if(lowertext(newletter)=="o")	newletter="u"
+				if(lowertext(newletter)=="s")	newletter="ch"
+				if(lowertext(newletter)=="a")	newletter="ah"
+				if(lowertext(newletter)=="c")	newletter="k"
```

Comment:
Recommending to in-line the `!skip_replacement`, and also some style cleanup while you're here.
```suggestion
		if(!skip_replacement && rand(1, 3) == 3)
			if(lowertext(newletter) == "o")	
				newletter = "u"
			if(lowertext(newletter) == "s")
				newletter = "ch"
			if(lowertext(newletter) == "a")
				newletter = "ah"
			if(lowertext(newletter) == "c")
				newletter = "k"
```